src/containers/SimpsonsQuote.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getLoading, getQuote } from '../selectors/simpsonsSelectors';
+import { setQuotePromise } from '../actions/simpsonsActions';
+import Quote from '../components/Quote';
+import Load from '../components/Load';
+import styles from './SimpsonsQuote.css';
+
+const SimpsonsQuote = () => {
+  const quote = useSelector(state => getQuote(state));
```

Comment:
This is supposed to be a class component where you use componentDidMount, mapStateToProps, and mapDispatchToProps